examples/notebooks/remoteRover.ipynb

```diff
@@ -194,12 +287,8 @@
    "metadata": {},
    "source": [
     "#!rover\n",
-    "\n",
-    "roverBrain.Act = (rover, now, token) => { \n",
-    "    if(currentSector != null){\n",
-    "        rover.TiltController.GoToSector(currentSector);\n",
-    "    }   \n",
-    "};"
+    "RoverBrain.ClearState();\n",
+    "CameraSector currentSector = null;"
```

Comment:
this is already declared above and reset to null to RoverBrain.ClearState(), right?

Reply:
removed, was a test leftover